src/odemis/acq/path.py

```diff
@@ -259,6 +260,28 @@
 ALIGN_MODES = {'mirror-align', 'chamber-view', 'fiber-align', 'streak-align', 'spec-focus', 'spec-fiber-focus', 'streak-focus'}
 
 
+class OneTaskExecutor(ThreadPoolExecutor):
+    """
+    Only one task at a time is executed. If several tasks are queued, only the
+    last one is executed.
+    """
+
+    def __init__(self, thread_name_prefix=''):
+        super(OneTaskExecutor, self).__init__(max_workers=1, thread_name_prefix=thread_name_prefix)
```

Comment:
doc string?

Reply:
It's just the same as the overriding method (- `max_workers`).